docs/rules/class-methods-use-this.md

```diff
@@ -0,0 +1,42 @@
+# Enforce that class methods utilize `this`. (class-methods-use-this)
+
+If a class method does not use this, it can safely be made a static function.
```

Comment:
Need a lot more in this section. Include some examples and explain what you're talking about. See http://eslint.org/docs/rules/no-undefined as an example.